src/main/java/com/cburch/logisim/gui/start/Startup.java

```diff
@@ -606,13 +577,28 @@ private static RC handleArgTemplate(Startup startup, Option opt) {
       logger.error(S.get("argOneTemplateError"));
       return RC.QUIT;
     }
-    final var file = opt.getValue();
-    startup.templFile = new File(file);
-    String errMsg = null;
-    if (!startup.templFile.exists()) errMsg = S.get("templateMissingError", file);
-    if (errMsg == null && !startup.templFile.canRead()) errMsg = S.get("templateCannotReadError", file);
-    if (errMsg != null) return RC.WARN;  // FIXME: shouldn't we quit in such case?
-    return RC.OK;
+    // first we get the option
+    final var option = opt.getValue();
+    // we look if it is a file
+    final var file = new File(option);
+    if (file.exists()) {
+      startup.templFile = file;
+      String errMsg = null;
```

Comment:
`errMsg` is not used. Also the conditions can be merged and simplified:

```java
      if (!startup.templFile.canRead()) {
        logger.error(S.get("templateCannotReadError", file));
        return RC.QUIT;
      }
      return RC.OK;
```

Reply:
done